paper.md

```diff
@@ -42,69 +42,77 @@ bibliography: paper.bib
+We use the hyperparameters reported by @spiekermann2022fast as implemented in the `barrier_prediction` branch, which is publicly available on [GitHub](https://github.com/kspieks/chemprop/tree/barrier_prediction) [@spiekermann_forked_chemprop]. <!-- We use the `barrier_prediction` branch from a forked version of Chemprop [@yang2019analyzing; @spiekermann_forked_chemprop] to train a deep message passing neural network using the hyperparameters reported by ref. [@spiekermann2022fast]. -->
 The QM9 dataset and RDB7 datasets were organized into 100 and 10 clusters, respectively.
-The results tabulated below show an expected trend that average model performance is worse (MAE and RMSE are higher) when faced with more challenging extrapolation tasks.
 
-### Average testing errors for predicting the 12 regression targets from QM9 [@ramakrishnan2014quantum].
+Table 1 and Table 2 show the expected trend in which the average testing errors are higher for the extrapolation tasks than they are for the interpolation tasks.
+The results from random splitting are informative if the model will be primarily used in interpolation settings. 
+However, these errors are unrealistically low if the model is intended to make predictions on new molecules. 
+Performance is worse on the extrapolative data splits, which present a more challenging task, but these errors should be more representative of evaluating a new sample.
+Together, these tables demonstrate the utility of `astartes` in allowing users to better understand the likely performance of their model in different settings.
+
+### Table 1: Average testing errors for predicting the 12 regression targets from QM9 [@ramakrishnan2014quantum].
 
 | Split     | MAE              | RMSE            |
 |-----------|------------------|-----------------|
-| random    | 2.02 $\pm$ 0.06  | 3.63 $\pm$ 0.21 |
-| scaffold  | 2.XX $\pm$ 0.XX  | 3.XX $\pm$ 0.XX |
+| Random    | 2.02 $\pm$ 0.06  | 3.63 $\pm$ 0.21 |
+| Scaffold  | 2.XX $\pm$ 0.XX  | 3.XX $\pm$ 0.XX |
 | K-means   | 2.XX  $\pm$ 0.XX | 4.XX $\pm$ 0.XX |
 
 
-###  Testing errors in kcal/mol for predicting a reaction's barrier height from RDB7 [@spiekermann2022high}.
+### Table 2: Testing errors in kcal/mol for predicting a reaction's barrier height from RDB7 [@spiekermann2022high].
 
 | Split     | MAE             | RMSE            |
 |-----------|-----------------|-----------------|
-| random    | 3.94 $\pm$ 0.03 | 6.89 $\pm$ 0.24 |
-| scaffold  | 4.YY $\pm$ 0.YY | 7.YY $\pm$ 0.YY |
+| Random    | 3.94 $\pm$ 0.03 | 6.89 $\pm$ 0.24 |
+| Scaffold  | 4.YY $\pm$ 0.YY | 7.YY $\pm$ 0.YY |
 | K-means   | 5.YY $\pm$ 1.YY | 7.YY $\pm$ 2.YY |
 
-In each case, the performance of the model is diminished when training and testing using extrapolative sampling algorithms.
```

Comment:
Rephrase this according to the discussion, explain how training and tuning in an interpolation scheme and then moving to usage with new without verifying that it is still interoplative can lead to unexpected errors, and that users can either verify their new data is still interopative (or else continue in the dark, technically should not eben use the models for extrpolation) or they can retrain with extrapolation in mind in the first place.

Reply:
Since the target audience of this paper is ML Scientists who are training models, I tried to shift the discussion of responsibility to the modelers so they are motivated to report performance in both interpolative and extrapolative tasks. Discussing downstream users and whether or not they use said models responsibly seemed trickier to incorporate but I welcome additional thoughts. The main point is that this analysis on both easy and more challenging splits should leave users better informed and hopefully encourage better adoption of these models. Happy to hear additional thoughts!